R/oi_active_cycle.R

```diff
@@ -0,0 +1,68 @@
+#' Function to recategorise OSM infrastrcutre based on cycleability
+#' 
+#' Determins whether or not a piece of OSM infrastructure is cycleable or not 
+#' depending on whether the feature is included in the default cycling network
+#' defined by [`osmextract`](https://github.com/ropensci/osmextract)
+#' 
+#' @usage oi_active_cycle(osm_sf, remove=FALSE)
+#' @param osm_sf - A `sf` and `data.frame` object containing OpenStreetMap 
+#'   infrastructure data, obtained from the 
+#'   [`osmextract`](https://github.com/ropensci/osmextract) function.
+#' @param remove - Boolean - If TRUE, features that do not form part of a 
+#'    cycleable network are removed from the `osm_sf` before being returned
+#' @return an sf object with oi_active_cycle column added.
+#' @details This function mimics the cycling filtering steps applied by the 
+#' [`oe_get_network`](https://rdrr.io/cran/osmextract/man/oe_get_network.html)
+#' function, part of the 
+#' [`osmextract`](https://rdrr.io/cran/osmextract/) package.
+#' @export oi_active_cycle
+#' @examples
+#' data = example_data
+#' example_output = oi_active_cycle(data)
+#' #plot(output["oi_cycle"], key.pos = 1)
+#' 
+"oi_active_cycle"
```

Comment:
I think this line and the next one can be deleted. You only need lines like this for the data docs I think.

Reply:
See [edf1a27](https://github.com/udsleeds/openinfra/commit/edf1a27e1b7d4174374b1c5227010ecb8cb6d570)